openapi.yaml

```diff
@@ -0,0 +1,79 @@
+openapi: 3.0.0
+info:
+  title: Web API Skeleton
+  description: Skeleton for Dropwizard Web APIs
+  version: '0.0'
```

Comment:
`version` here comes from this:
https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#user-content-infoVersion
> REQUIRED. The version of the OpenAPI document (which is distinct from the OpenAPI Specification version or the API implementation version).

I think it would make more sense for it to be `'1.0'` though so I'll change it